### PR TITLE
Add zip code to API key request form

### PR DESCRIPTION
**Pull request by shaisachs**

### CODE REVIEW

File: docs/action-tag/usage/overview.jade [hunk 1]
```diff
@@ -20,21 +20,13 @@ block description
...
-  script(src='//d1aqhv4sn5kxtx.cloudfront.net/nvtag.js')
-  div(
-    class='ngp-form'
-    data-id='-9219430287669198848'
-    data-databag='everybody'
-    data-formdef-endpoint='//formdefs.s3.amazonaws.com/api.myngp.com'
-    data-template='minimal'
-  )
+    **July 2015 Update:** we have released major changes to ActionTag. The primary effect is that consumers of ActionTag need to update the js URL from `//d1aqhv4sn5kxtx.cloudfront.net/nvtag.js` and instead use the following url: `//d1aqhv4sn5kxtx.cloudfront.net/actiontag/at.js`. The markup and CSS is new, in order to be more in line with our ProgressUI library. Our callback structure is unchanged.
```
Comment (shaisachs) [inline docs/action-tag/usage/overview.jade]: This should be listed in changelog.

Comment (rwinikates) [inline docs/action-tag/usage/overview.jade]: Fair point, but where do we keep said changelog?

Comment (shaisachs) [inline docs/action-tag/usage/overview.jade]: Just in the root of the repo at CHANGELOG.md

Comment (rwinikates) [inline docs/action-tag/usage/overview.jade]: done

**Pull request merged**